source/Octopus.Tentacle.Client/TentacleClient.cs

```diff
@@ -94,7 +94,7 @@ TentacleClientOptions clientOptions
             rpcCallExecutor = RpcCallExecutorFactory.Create(this.clientOptions.RpcRetrySettings.RetryDuration, this.tentacleClientObserver);
         }
 
-        public TimeSpan OnCancellationAbandonCompleteScriptAfter { get; set; } = TimeSpan.FromMinutes(1);
+        public TimeSpan OnCancellationAbandonCompleteScriptAfter { get; set; } = TimeSpan.FromMinutes(5);
```

Comment:
This timeout should only apply when the ServerTask is cancelled. The customer's TaskLog indicates it wasn't cancelled so either there is a bug somewhere else in the code making this apply to all requests or it is another issue.

Reply:
I would suggest bumping this to 5 mins is not the fix we want here

Reply:
The bug is in TentacleClient

![image](https://github.com/OctopusDeploy/OctopusTentacle/assets/86938706/acb9a22b-8f1c-4a90-b245-42b7dec94a0b)

Cancellation after 1 minute is being applied regardless of if the `scriptExecutionCancellationToken` has been cancelled.

Reply:
SaST should be able to pick this up under warranty, as the bug was introduced relatively recently.

Reply:
cc @LukeButters

Reply:
Bug introduced here https://github.com/OctopusDeploy/OctopusTentacle/pull/730/files?w=1#diff-220f6bbde952d88b09ae1c1e9e46064e97134a455c41ad7b05e0aafb333c652cR215